modules/payments_stripe/public/schema/payment_method.yml

```diff
@@ -0,0 +1,23 @@
+properties:
```

Comment:
Don't you need the name of the model in yml file?

Reply:
no, it is taken from file name